uvicorn/loops/auto.py

```diff
@@ -1,11 +1,14 @@
-def auto_loop_setup() -> None:
+from typing import Any
+
+
+def auto_loop_setup(**kwdargs: Any) -> None:
```

Comment:
can't we use an explicit reload argument here and below or there is something I dont get ?